js/fetch.ts

```diff
@@ -399,6 +401,28 @@ export async function fetch(
     headers = input.headers;
   }
 
+  // Fetching a local file
+  const m = url.match(/file:\/\/\/(.*)/i);
+  if (m && m[0]) {
+    try {
+      const f = openSync(m[1]);
+      const headers: [string, string][] = [];
+      // headers.push(["content-type", contentType(extname(m[1])) || "text/plain"])
+      headers.push(["content-type", "text/plain"]);
```

Comment:
I think for now you don't need to return a content-type. It would be ideal if we could import https://github.com/denoland/deno_std/tree/master/media_types but that's not possible at the moment. Could you add a TODO comment that mentions this?

Reply:
Ok i was into the same approach. Sure i'll add it.